src/components/Icon/index.tsx

```diff
@@ -20,13 +21,14 @@ const Icon: React.FC<IIconProps> = ({
   const SvgContent = SvgRoot[name];
 
   return (
-    <SvgContent
-      width={width}
-      height={height || width}
-      viewBox={'0 0 48 48'}
-      fill={color}
-      {...rest}
-    />
+    <View {...rest}>
```

Comment:
Since we pass SvgProps to Icon component, I think `{...rest}` should be passed to SvgContent instead of View

Reply:
Yeah u are right